### DISCUSSION TITLE
HELP! Need version 5.0 installer for Mac, because the 5.6 won't open on Catalina

### DISCUSSION THREAD

**Discussion by WildcatSound** [Anjok07/ultimatevocalremovergui#1116] [Q&A]
Hi guys, i was working fine with an ond version of UVR (i guess it was the 5.0 version, or the one for Catalina and above).
Yesterday i tried to upgrade to 5.6 but once it installed then never open. I saw on discussions many guys had this problem but i couldnt fix it.
Im looking for the older installer for version 5.0 i cant find it nowhere . Can somebody kindly send me pls?
I run Os Catalina 10.15.7 on Macbook pro 2,5 GHz Intel Core i5 dual-core
thank you

**Comment by aleprogressive** [comment]
Same here.
